wg-meetings/2016-06-01.md

```diff
@@ -0,0 +1,91 @@
+# Node.js Tracing/Diag WG Meeting
+
+- Date: 2016-06-01
+- YouTube: http://www.youtube.com/watch?v=tNxYDvQ1kT8
+
+## Attendees
+
+- @joshgav
+- @mhdawson
+- @matthewloring
+- @ofrobots
+- @paulirish
+- @pavelfeldman
+- @caseq
+- @jasonlaster
+- @pmuellr
+- @fmeawad
++ several more via YouTube
+   
+## Agenda
+
+- TRACE_EVENT - Google's plans and how Node can build on them (@ofrobots)
+- Trace module - @joshgav's proposal - why and what.
+    - [nodejs/tracing-wg#50], [joshgav/node-trace]
+- Chrome Debugging Protocol and v8_inspector (@ofrobots, Google)
+    - [nodejs/node#6792], [nodejs/node#7072]
+    - What it brings to Node and the community now and in the future.
+    - How can the Diagnostics WG utilize it?
+- AsyncWrap [deferred]
+- Expanding scope of this WG [deferred]
+
+## Next Meeting
+
+- Return to monthly cadence - first Wednesday of each month at 12pm US Pacific (immediately before CTC meeting). Next meeting will be July 6.
+
+## Minutes
+
+### Tracing
+
+In Chromium, the Trace Event trace system is implemented through a Tracing Controller. @ofrobots and @matthewloring are working on porting this implementation to V8/D8.
```

Comment:
Actually, @fmeawad and team have been working on porting this to V8/D8. Once this is ready, @matthewloring and I are planning to start working on extending this and bring a Tracing Controller to Node.